app/clientsHandler.js

```diff
@@ -0,0 +1,78 @@
+const { idGenerator } = require("./utils.js");
+
+const generateUniqueClientId = idGenerator();
+
+const CLIENTS_HANDLER = (() => {
+
+    // Client objects in general
+    const CLIENTS = {};
+    // Registered client ids
+    const REGISTERED_CLIENTS = new Set();
+    // Registered client ids indexed by game and username
+    const REGISTERED_CLIENTS_BY_GAME = {};
+
+    /**
+     * 
+     * @param {string} clientId 
+     * @param {string} username 
+     * @param {string} gameId
+     * This method registers a clientId to a specific game
+     */
+    const registerClient = (clientId, username, gameId) => {
+        REGISTERED_CLIENTS.add(clientId);
```

Comment:
We shouldn't use clientId, either. If you close your browser and try to rejoin the game, you will have a different clientId because you have a new Websocket connection. Let's use tokens instead! We should talk about this